src/RumpelWalletFactory.sol

```diff
@@ -45,6 +45,9 @@ contract RumpelWalletFactory is Ownable, Pausable {
         uint256 threshold,
         InitializationScript.InitCall[] calldata initCalls
     ) external whenNotPaused returns (address) {
+        // Calculate a unique salt based on the sender's address and nonce.
+        bytes32 salt = keccak256(abi.encodePacked(msg.sender, saltNonce[msg.sender]++));
```

Comment:
nit - can you cast to a uint here instead of later?

Reply:
https://github.com/sense-finance/rumpel-wallet/pull/6/commits/28202ca6e0282d54ac150bd1ef68d254abbaf082